meilisearch-release-changelog/v1.9.0.md

```diff
@@ -0,0 +1,187 @@
+# v1.9.0 release changelogs
+
+<!-- The following line should ONLY be put PRE-release changelogs -->
+⚠️ Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Is something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+Meilisearch v1.9 includes performance improvements for hybrid search and the addition/updating of settings. This version benefits from multiple requested features, such as the new `frequency` matching strategy and the ability to retrieve similar documents.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after a new version becomes available.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+Some SDKs might not include all new features. Consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that ❤️).
+
+# New features and updates 🔥
+
+## Hybrid search updates
+
+This release introduces multiple [hybrid search updates](https://meilisearch.notion.site/v1-9-AI-search-changes-e90d6803eca8417aa70a1ac5d0225697#38e6d3adf40e4ef1be14a3c4be39df94).
+
+Done by @dureuill and @irevoire in #4633 and #4649
+
+### ⚠️ Breaking change: Empty `_vectors.embedder` arrays
+
+Empty `_vectors.embedder` arrays are now interpreted as having no vector embedding. 
+
+Before v1.9, Meilisearch interpreted these as a single embedding of dimension 0. This change follows user feedback that the previous behavior was unexpected and unhelpful.
+
+### ⚠️ Breaking change: `_vectors` field no longer present in search results
+
+Meilisearch no longer includes `_vectors` in returned search results by default. This will considerably improve performance.
```

Comment:
```suggestion
When the experimental `vectorStore` feature is enabled, Meilisearch no longer includes `_vectors` in returned search results by default. This will considerably improve performance.
```